MovieQueueManagerGUI/MovieQueueManagerGUI.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Almost anything that happens in *.csproj files is auto generated. When I upgraded this project to .net 4.6.1 it also updated the version of MSBuild that it uses to build the project. The schema is referring to what elements are allowed/expected in an XML file. These two links may help give you some background: [general info about xmlns](http://www.w3schools.com/xml/xml_namespaces.asp) and [specifically how Microsoft uses it](https://msdn.microsoft.com/en-us/library/ms754619(v=vs.85).aspx)